client/modules/IDE/components/ConsoleInput.jsx

```diff
@@ -24,75 +25,110 @@ function ConsoleInput({ theme, dispatchConsoleEvent, fontSize }) {
       inputStyle: 'contenteditable'
     });
 
-    cmInstance.current.on('keydown', (cm, e) => {
+    cmInstance.current.getWrapperElement().style['font-size'] = `${fontSize}px`;
+  }, []);
+
+  useEffect(() => {
+    if (cmInstance.current) {
+      cmInstance.current.setOption('theme', `p5-${theme}`);
+      cmInstance.current.getWrapperElement().style[
+        'font-size'
+      ] = `${fontSize}px`;
```

Comment:
I think we can add a `style` property to one of the `div`s that we control instead of manipulating the element.

```
<div ref={codemirrorContainer} className="console__editor" style={{ fontSize: `${fontSize}px` }} />
```

Where it is being set currently is a direct child of that div but it has the same effect.

Reply:
Just tried it out!